src/test/shell/bazel/bazel_java14_test.sh

```diff
@@ -125,7 +125,8 @@ public class Javac14Example {
   }
 }
 EOF
-  bazel run java/main:Javac14Example --test_output=all --verbose_failures &>"${TEST_log}"
+  # TODO(ilist): remove tool_java_runtime_version after java_runtime is attached to the toolchain
```

Comment:
Is there an issue to attach to this?

Reply:
No issues. I wanted to keep PR smaller, it is already quite big.